llvm/lib/Target/DirectX/DXILOpLowering.cpp

```diff
@@ -259,6 +259,114 @@ class OpLowerer {
       lowerToBindAndAnnotateHandle(F);
   }
 
+  /// Replace uses of \c V with the values in the `dx.ResRet` of \c Op. Since we
```

Comment:
What is `V` in this context?

Reply:
Nice catch, should be "Intrin", the name of the first parameter. Updated.